app/build.gradle

```diff
@@ -55,5 +59,5 @@ dependencies {
     // optional - Test helpers
     testImplementation "androidx.room:room-testing:$room_version"
 
-    androidTestImplementation 'androidx.test:rules:1.1.0'
```

Comment:
don't remove this, makes tests not run

Reply:
we just moved it to the top to not make it stand out anymore